docs/_source/_templates/versions.html

```diff
@@ -0,0 +1,42 @@
+{% if GHPAGES %}
+    <div class="rst-versions" data-toggle="rst-versions" role="note" aria-label="versions">
+    <span class="rst-current-version" data-toggle="rst-current-version">
+      <span class="fa fa-book"> Read the Docs</span>
+      v: {{ version }}
+      <span class="fa fa-caret-down"></span>
+    </span>
+
+        <div class="rst-other-versions">
+            <dl id="other-versions">
+                <dt>Versions</dt>
+            </dl>
+            <dl>
+                {% set links = {
+                'Cloudreach':'https://www.cloudreach.com',
+                'Sceptre Home': 'https://www.sceptre.cloudreach.com'
+                } %}
+                <dt>Other Links</dt>
+                {% for title, url in links.items() %}
+
+                    <dd>
+                        <a href="{{ url }}">{{ title }}</a>
+                    </dd>
+                {% endfor %}
+            </dl>
+            <hr/>
+            Free document hosting provided by <a href="http://www.readthedocs.org">Read the Docs</a>.
```

Comment:
Can remove this attribution as we are not hosting on RTD.

Reply:
Remove line entirely or change the reference to:

`Original template provided by <RTD>..` ?

Reply:
Good idea - keep the attribution by change it to what you suggested.